src/formatDate.js

```diff
@@ -50,7 +50,31 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const newArr = date.split(`${fromFormat[fromFormat.length - 1]}`);
+  const tempArr = [];
+
+  for (let i = 0; i < newArr.length; i++) {
+    if (i !== newArr.length) {
+      for (let j = 0; j < newArr.length; j++) {
+        if (fromFormat[i] === toFormat[j]) {
+          tempArr[j] = newArr[i];
+        } else if ((fromFormat[i] === 'YY' || fromFormat[i] === 'YYYY')
+        && (fromFormat[i] !== toFormat[j])) {
+          if (fromFormat[i] === 'YY' && toFormat[j] === 'YYYY') {
```

Comment:
this, and 69-71 rows
![image](https://github.com/mate-academy/js_format_date_advanced/assets/86923089/05a17e11-853a-4b58-8a9d-9970674f7e21)

might be handled in a bit different way

try to overwrite it (you can use an object to store your values, etc)